config/bash/completion/cobbler

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
Please revert. On Debian `bash` is in `/usr/bin/`:
```
# which bash
/usr/bin/bash
```

Reply:
Ah now we got a problem... The buildservice doesn't like `env`. I will try to think of something to get around `env` and do it correctly on both sides...

Reply:
I fixed it via a patch which just gets applied on the debian based build.